src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -116,6 +121,31 @@ public UserRole getUserRoleClassifications(String userRole) {
         }
     }
 
+    @Override
+    public List<Jurisdiction> getAllJurisdictions() {
+        try {
+            LOG.debug("retrieving all jurisdictions definition");
+            final HttpEntity requestEntity = new HttpEntity(securityUtils.authorizationHeaders());
+            UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(applicationParams.jurisdictionDefURL());
+            return restTemplate.exchange(builder.build().encode().toUri(), HttpMethod.GET,
+                    requestEntity, new ParameterizedTypeReference<List<Jurisdiction>>() {}).getBody();
+        } catch (Exception e) {
+            LOG.warn("Error while retrieving jurisdictions definition", e);
+            if (e instanceof HttpClientErrorException
+                    && ((HttpClientErrorException)e).getRawStatusCode() == RESOURCE_NOT_FOUND) {
+                throw new ResourceNotFoundException("Resource not found when retrieving jurisdictions definition because of " + e.getMessage());
+            } else {
```

Comment:
Checkstyle should complain about line length

Reply:
I know but we agreed line length of 120 chars is a bit too little so sometimes I ignore it
I really think with nowadays wide screens 120 chars is too small and the line wrapping that causes makes the code even less readable!

Reply:
It is now open sourced and well